mlflow/utils/uri.py

```diff
@@ -126,6 +127,12 @@ def is_databricks_unity_catalog_uri(uri):
     return scheme == _DATABRICKS_UNITY_CATALOG_SCHEME or uri == _DATABRICKS_UNITY_CATALOG_SCHEME
 
 
+# Update scheme parsing if changes are made to the OSS URI (this is an initial proof of concept)
```

Comment:
Update or remove this comment?  It's not a POC anymore. 🙂

Reply:
done